pyogrio/geopandas.py

```diff
@@ -74,9 +74,11 @@ def read_dataframe(
         Number of features to read from the file.  Must be less than the total
         number of features in the file minus skip_features (if used).
     where : str, optional (default: None)
-        Where clause to filter features in layer by attribute values.  Uses a
-        restricted form of SQL WHERE clause, defined here:
-        http://ogdi.sourceforge.net/prop/6.2.CapabilitiesMetadata.html
+        Where clause to filter features in layer by attribute values. If the datasource
+        natively supports sql, its specific sql dialect should be used (eg. SQLite and
```

Comment:
```suggestion
        natively supports SQL, its specific SQL dialect will be used by default (eg. SQLite and
```

Reply:
I think this should not be changed, as it is not only by default, but the user cannot change it, so the only option is that he uses the SQLITE dialect in this instance